### ISSUE TITLE
How to show key (legend) for scaled markers

### ISSUE THREAD

**Issue by wb11n** [veusz/veusz#434]
In my xy widget, markers for plotted data are scaled by size with respect to a third variable.  I know there is a colour bar for showing variation in colour, is there something available for size?? I would be happy to use this widget.

**Comment by jeremysanders** [comment]
No, there's nothing currently included. You could manually add something similar by adding a new point plotter widget to plot some some markers yourself. You could use e.g. "1,2,3,4" as your y dataset, "1,1,1,1" as x dataset, "1,2,3,4" as your scale values. If you want them labelled, you'll need to create a text dataset and put this as labels.

**Comment by wb11n** [comment]
Thanks Jeremy. I think this should work out just fine, and would require manual touches (e.g., to display it outside the plot window so it doesn't overlap with the main data). Thanks for the effort you put into this project. I am very new to Veusz but it's given impressive results so far. 
One feature I would love to see in future releases is to be able to have grouped scatter plots, where data can be visualised into groups/categories based on size or colour indexing. Forgive me if this feature is available already.
